frontend/src/components/camp/Picasso.vue

```diff
@@ -245,6 +229,10 @@ export default {
     },
     timeMouseDown (tms) {
       const mouse = this.toTime(tms)
+      if (this.openedInNewTab) {
+        this.openedInNewTab = false
+        return
+      }
```

Comment:
Can you explain to me why this is necessary?

Reply:
Prevents `showScheduleEntry` from being triggered which would open the activity in the current tab as well (or the popup, previously). If you know a better way to achieve this let me know, but that's what I came up with as a Vue beginner :-)

Reply:
I see it now, I forgot about the confusing behaviour of VCalendar that triggers both `eventMouseDown` and `timeMouseDown`. Maybe we can do better if we plug a 100% width 100% height `div` into each event that can catch and stop `click` events. But for now this will do.